generate-bookmarks-html

```diff
@@ -96,9 +96,8 @@ EOD
             fi
             if [[ "$line" =~ ^\*\ @([a-z]*)\ (.*) ]]
             then
-                tag="${BASH_REMATCH[1]}"
```

Comment:
I can assure you this extra variable is not the reason the script is so slow. :-) And even I use a variable just once, I tend to keep it just for code clarity/understandability.